base/applications/regedit/childwnd.c

```diff
@@ -473,9 +466,6 @@ LRESULT CALLBACK ChildWndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lPa
         }
         break;
 
-    case WM_TIMER:
-        break;
-
```

Comment:
Is that is some ancient left-over of a timer that is no longer present?

Reply:
There is no `SetTimer` call. It's meaningless.

```bash
$ git blame base/applications/regedit/childwnd.c
...
a0f2e6b8cacd rosapps/regedit/childwnd.c                   (Robert Dickenson      2002-07-14 21:03:10 +0000 476)     case WM_TIMER:
a0f2e6b8cacd rosapps/regedit/childwnd.c                   (Robert Dickenson      2002-07-14 21:03:10 +0000 477)         break;
...
```